File: laravel-demo/app/Http/Controllers/Pages/ForumsController.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class ForumsController extends Controller
{
    public function discussion()
    {
        return view('pages.forums.discussion', ['title' => 'Forum discussion - Pages']);
    }

    public function list()
    {
        return view('pages.forums.list', ['title' => 'Forum list - Pages']);
    }

    public function threads()
    {
        return view('pages.forums.threads', ['title' => 'Forum threads - Pages']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/Pages/ForumsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers.Pages
{
    public class ForumsController : Controller
    {

         public ActionResult Discussion()
        {
            return View("~/Views/Pages/Forums/Discussion.cshtml");
        }

         public ActionResult List()
        {
            return View("~/Views/Pages/Forums/List.cshtml");
        }

         public ActionResult Threads()
        {
            return View("~/Views/Pages/Forums/Threads.cshtml");
        }

    }
}
